src/packages/sdk/src/getGenerators.ts

```diff
@@ -543,25 +543,7 @@ export function skipIndex<T = any>(arr: T[], index: number): T[] {
   return [...arr.slice(0, index), ...arr.slice(index + 1)]
 }
 
-export const knownBinaryTargets: Platform[] = [
-  'native',
-  'darwin',
-  'darwin-arm64',
-  'debian-openssl-1.0.x',
-  'debian-openssl-1.1.x',
-  'linux-arm-openssl-1.0.x',
-  'linux-arm-openssl-1.1.x',
-  'rhel-openssl-1.0.x',
-  'rhel-openssl-1.1.x',
-  'linux-musl',
-  'linux-nixos',
-  'windows',
-  'freebsd11',
-  'freebsd12',
-  'openbsd',
-  'netbsd',
-  'arm',
-]
+export const knownBinaryTargets: Platform[] = [...platforms, 'native']
```

Comment:
This is neat!

Note platforms is defined here https://github.com/prisma/engines-wrapper/blob/master/packages/get-platform/src/platforms.ts